src/data/community-events.json

```diff
@@ -484,5 +502,14 @@
     "description": "The Ethereum dev conference in Dubai on everything DeFi, NFTs, EVM scaling, layers 2 and more with a focus on decentralization and community projects.",
     "startDate": "2023-03-14",
     "endDate": "2023-03-16"
+  },
+  {
+    "title": "ETH PRIVACY",
```

Comment:
What is the name of the event? This isn't very clear from the website. Either way, I imagine this doesn't need to be capitalised.